src/imessage/identity_manager.rs

```diff
@@ -268,8 +286,8 @@ impl Resource for IdentityResource {
             })
         }
         debug!("Register success!");
-
-        self.cache.lock().await.verity(&self.aps, &users_lock).await;
```

Comment:
I was unable to figure out how to make this work properly, or if the `verity ` function was even necessary?